wordpress-mobile/WordPress-Android
# Issue/4401 direct reply notif

Fixes #4401 

I'm chaining this PR to #4667 as we need the service to implement this one.

![notif_direct_reply](https://cloud.githubusercontent.com/assets/6597771/19571829/eafccfd0-96d5-11e6-9b3f-ccd9fa071b97.gif)


To test:
1. Log in to the Android WordPress app on an Android 7.x device or emulator
2. Have someone like/comment on a blog post of yours
3. See the notification arrive at your device
4. Click on the "REPLY" quick action
5. the edittext appears where you can type the reply
6. tap the SEND button
7. watch the comment being sent and check the comment arrives at the other end (i.e. is received by the first commenter)

cc @kwonye
